app/javascript/components/miq-data-table/index.jsx

```diff
@@ -100,9 +100,11 @@ const MiqDataTable = ({
         isSortable={isSortable}
         isSortHeader={sortHeader}
         sortDirection={sortDirection}
-        className={
-          classNames('miq-data-table-header', (header.contentButton ? 'header-button' : ''), (header.actionCell ? 'action-cell-holder' : ''))
-        }
+        className={classNames(
+          'miq-data-table-header', 
```

Comment:
@liu-samuel can you please fix the lint errors in this file. On lines 104 and 105 there is an extra space at the end of the lines. Can you please remove the extra space and repush.